libraries/botbuilder-dialogs/tests/choices_channel.test.js

```diff
@@ -98,4 +99,15 @@ describe('channel methods', function() {
         const channel = getChannelId({ activity: { } });
         assert(channel === '', `expected "", instead received ${ channel }`);
     });
+
+    // "directlinespeech" tests
+    it(`should return true for supportsSuggestedActions() with directlinespeech and 100`, function () {
+        const validNumOfSuggestedActions = supportsSuggestedActions(Channels.DirectlineSpeech, 100);
+        assert(validNumOfSuggestedActions, `returned false.`);
```

Comment:
`returned true.`?

I am very confused since other tests have similar issues.

Reply:
The `assert` used here is an alias for `assert.ok` which performs a truthy evaluation; the "returned false." is a custom error message when the `assert` fails.

These tests should be refactored, but that's out of this PR's scope and in @axelsrz's Code Quality pillar. I'll coordinate with him on updating these and other tests.